src/requestHandler.ts

```diff
@@ -70,7 +70,7 @@ const getLDRequestHandler = (sdkKey: string, getLDUser: GetLDUser) => {
     }
 
     if (
-      [/^\/_next/, /^\/static/, /^\/public/, /\.\w+$/].find((matcher) =>
```

Comment:
Just realized wouldn't dropping ` /\.\w+$/` be sufficient?

Reply:
yeah I did a bit more than necessary


- static is a convention of our app and it does not seem like i18next still loads the jsons over the netork
- public does not actually ever appear in the path